.eslintrc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     'react-hooks/rules-of-hooks': ERROR,
     'react/display-name': WARNING,
     'react/jsx-no-bind': IGNORE,
+    'react/jsx-pascal-case': IGNORE,
```

Comment:
where are we breaking this rule?

Reply:
it's mad about `Text.P`. seems like a bug in the plugin

Reply:
This fixes itself if you run npm i